homeassistant/helpers/network.py

```diff
@@ -165,6 +166,7 @@ def get_url(
                     require_current_request=require_current_request,
                     require_ssl=require_ssl,
                     require_standard_port=require_standard_port,
+                    require_cloud=require_cloud,
```

Comment:
I don't think we should be able to combine the require cloud option with other options.